packages/gatsby-theme-newrelic/src/utils/page-tracking/segment.js

```diff
@@ -0,0 +1,67 @@
+import { AnalyticsBrowser } from '@segment/analytics-next';
+import warning from 'warning';
+import { getResolvedEnv, getSegmentConfig } from '../config';
+import Cookies from 'js-cookie';
+import { DEV_SEGMENT_WRITE_KEY } from '../constants';
+
+const warnAboutNoop = (section, platform) => {
+  warning(
+    section && platform,
+    "You have enabled page view tracking, but do not have page view tracking configured. This route change has not been tracked. Please configure the 'segment.section' and 'segment.platform' options in gatsby-config.js"
+  );
+};
+
+const trackPageView = ({ config, env, location, prevLocation, analytics }) => {
+  const { section, platform } = config;
+  const getCookie = (key) => Cookies.get(key)?.replace(/%22/g, '') || null;
+  const customer_user_id = getCookie('ajs_user_id');
+  const anonymousId = analytics.instance?.user().anonymousId();
```

Comment:
We're using a cookie for this one: `getCookie('ajs_anonymous_id');` in the tessen sunset PR. Do they need to match or nah?

Reply:
i'm not sure 😓 , this call will get the anonymous id for a user if it exists and otherwise create a new one

Reply:
oh that makes sense, this was set by segment anyway i believe